### DISCUSSION TITLE
Help TypeError when .use(createPinia()) vue3 with TypeScript

### DISCUSSION THREAD

**Discussion by joseluisgs** [vuejs/pinia#591] [Help and Questions] [answered]
Hi
I am using Pinia in Vue3 project with TypeScript to compare with Vuex and I have a problem when run de app with when trye tu use(createPinia()).
I have the following error in console:
Uncaught TypeError: Object(...) is not a function
    at createPinia (pinia.esm-bundler.js?77ba:845)
    at eval (main.ts?bc82:15)
    at Module../src/main.ts (app.js:1462)
    at __webpack_require__ (app.js:854)
    at fn (app.js:151)
    at Object.1 (app.js:1595)
    at __webpack_require__ (app.js:854)
    at checkDeferredModules (app.js:46)
    at app.js:994

I attach capture
![Captura de pantalla 2021-07-30 a las 13 17 48](https://user-images.githubusercontent.com/47913953/127647082-86e3d349-ae22-4fc7-a4a4-5e4757323f76.png)

this is my repo
https://github.com/joseluisgs/vue-kanban/tree/feature/firebase

This is the commit hash: eaffd79d56c07f254185c6d4816c4117b02b5a74

I will really appreciate the help to learn how to use Pinia without errors and to be able to enjoy using it.

**Comment by joseluisgs** [comment]
Ok, I change vue to "vue": "^3.2.0-beta.7" and "@vue/compiler-sfc": "^3.2.0-beta.7" and now works
Ok, a stupid question
If I use the default Vue Cli Template and dependences 
"vue": "^3.0.0", and "@vue/compiler-sfc": "^3.0.0", what is the version of Pinia that I need to use?
or, I dont't want to use a Beta version of Vue, whats is the most estable versio of Vue and Pinia to work together without problems?
Thanks!!!

**Comment by posva** [comment, marked as answer]
Follow https://github.com/posva/pinia/issues/592

**Reply by joseluisgs** [reply]
Thank you so much!!!!
